lisk/scripts/update_config.js

```diff
@@ -286,11 +292,29 @@ history.version('1.6.0-rc.0', version => {
 		config = moveElement(config, 'forging', 'modules.chain.forging');
 		config = moveElement(config, 'syncing', 'modules.chain.syncing');
 		config = moveElement(config, 'loading', 'modules.chain.loading');
+		return config;
+	});
 
-		// Future network module
-		config = moveElement(config, 'peers', 'modules.chain.network');
-		config = moveElement(config, 'wsPort', 'modules.chain.network.wsPort');
-		config = moveElement(config, 'address', 'modules.chain.network.address');
+	version.change('add structure for network module', config => {
+		config = moveElement(config, 'peers.list', 'modules.network.seedPeers');
+		config = moveElement(
+			config,
+			'peers.access.blackList',
+			'modules.network.blacklistedPeers'
+		);
+		config = moveElement(
+			config,
+			'peers.options.timeout',
+			'modules.network.connectTimeout'
+		);
```

Comment:
You also renamed `httpServerSetTimeout` to `ackTimeout`, Please add an entry for this as well.

Reply:
actTimeout and httpServerSetTimeout is not related, and httpServerSetTimeout was not used